### DISCUSSION TITLE
ScrollArea: default style ?

### DISCUSSION THREAD

**Discussion by cnouguier** [quasarframework/quasar#17391] [General - Components / Directives / etc]
Are there any possibilities to set a default style to the ScrollArea component? The idea is to ensure that an application has the same style when rendering scroll areas on different components. We could wrap the ScrollArea component but it could be nice if Quasar can handle this, such as the Notify plugin defaults.

**Comment by ivanjaros** [comment]
you can just use css.
if you want to style the component, you have to wrap it because you can alter only props, not attributes like css class.

**Reply by cnouguier** [reply]
Thanks for your suggestion.  We shall wrap the component.
